workspaces/src/network/account.rs

```diff
@@ -17,6 +24,60 @@ impl Account {
     pub(crate) fn signer(&self) -> &InMemorySigner {
         &self.signer
     }
+
+    /// Call a contract on the network specified within `worker`, and return
+    /// a Builder object that we will make use to populate the rest of the
+    /// call details.
+    pub fn call<'a, T: Network>(
+        &self,
+        worker: &'a Worker<T>,
+        contract_id: AccountId,
+        function: String,
+    ) -> CallBuilder<'a, T> {
+        CallBuilder::new(worker, contract_id, self.signer.clone(), function)
+    }
+
+    /// Transfer near to an account specified by `receiver_id` with the amount
+    /// specified by `amount_yocto`. Returns the execution details of this
+    /// transaction
+    pub async fn transfer_near<T: Network>(
+        &self,
+        worker: &Worker<T>,
+        receiver_id: AccountId,
+        amount_yocto: Balance,
```

Comment:
`Balance` is the type that gets used everywhere for this even such as `api-js`. It make sense for it to be balance since you are transferring `Balance`s between accounts. The name `Near` is already a bit bloated in the ecosystem is the thing. How about I just have it be `amount: Balance`? - since `Balance` is by default yocto and the expectation would be that a user use one of the utility functions such as `pase_near!("10 N")`